docs-site/docs/user/organization/members/members-intro.md

```diff
@@ -0,0 +1,9 @@
+---
+id: introduction
+---
+
+# Members
+
+In IRIDA Next, groups and projects can both have members with roles that are either directly added or inherited from parent groups.
+
+For more information about adding group and project members, see [Group and Project Members](members)
```

Comment:
This link is broken, if you fix the `id` in `membersship.md` to `members` it should work.

Reply:
Fixed in a63a4d4ad7b141bcf3f735f6b628037c06a95b84